lambda/src/new_account_iam_role.py

```diff
@@ -108,14 +107,15 @@ def get_session(assume_role_arn):
         assume_role_arn,
         RoleSessionName=generate_lambda_session_name(function_name),
         DurationSeconds=3600,
+        validate=False,
```

Comment:
If set to true, it will create a sts client, without an endpoint of our choosing.  It's purpose:

```
By default, the parameters are checked so that errors can be raised
at this point, rather than more confusingly when the first call
is made using the child session.
This can be disabled by setting validate=False.
```

Reply:
Ahh, it's doing validation by using an sts client. Interesting. Gotcha.